apps/common/src/main/scala/org/lfdecentralizedtrust/splice/store/UpdateHistory.scala

```diff
@@ -2174,6 +2174,31 @@ class UpdateHistory(
 }
 
 object UpdateHistory {
+
+  // Separate method so we can use this without a full UpdateHistory instance.
+  def getHighestKnownMigrationId(
```

Comment:
add a comment that we deliberately do not filter by store id because we run this too early in the setup where we don't have easy access to that.